System.IO.Ports/SerialData.cs

```diff
@@ -8,16 +8,29 @@ namespace System.IO.Ports
     /// <summary>
     /// Specifies the type of character that was received on the serial port of the <see cref="SerialPort"/> object.
     /// </summary>
+    /// <remarks>
+    /// This enumeration is used with the <see cref="SerialPort.DataReceived"/> event. 
+    /// You examine the type of character that was received by retrieving the value of the <see cref="SerialDataReceivedEventArgs.EventType"/> property.
+    /// The EventType property contains one of the values from the `SerialData` enumeration.
+    /// </remarks>
     public enum SerialData
     {
         /// <summary>
         /// A character was received and placed in the input buffer.
         /// </summary>
         Chars = 1,
 
+        // /// <summary>
```

Comment:
Why is this needed here?

Reply:
We should really support that `End of File`... But the current enum field was taken, and it is a catch 22 between removal and change...

Reply:
👀  https://social.msdn.microsoft.com/Forums/vstudio/en-US/6aaf1c70-46c6-485b-a769-2389da7684ee/ioportsserialport-datareceived-eventtypeeof

Reply:
Interesting... But that does not say we should not support it... Just fix it... But that is not the reason for this PR. If prefered, I can remove it, but feel it currently serves as a valid placeholder (given possibly reserved enum field).

Reply:
I don't think we are able to support EOF... what's your use case for it?

Reply:
After reading https://stackoverflow.com/questions/12483711/serialdata-eof-circumstances I agree that it is not something we should not support. I will remove it.